src/openzaak/components/besluiten/models/tests/test_bio_block_change.py

```diff
@@ -0,0 +1,47 @@
+from django.test import TestCase
+
+from vng_api_common.constants import RelatieAarden
+
+from openzaak.components.documenten.models.tests.factories import (
+    EnkelvoudigInformatieObjectFactory,
+)
+
+from ...models import BesluitInformatieObject
+from .factories import BesluitFactory, BesluitInformatieObjectFactory
+
+
+class BlockChangeTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.bio = BesluitInformatieObjectFactory.create()
+
+    def test_update(self):
+        self.assertRaises(
+            TypeError,
+            BesluitInformatieObject.objects.update,
+            aard_relatie=RelatieAarden.hoort_bij,
+        )
+
+    def test_delete(self):
+        self.assertRaises(TypeError, BesluitInformatieObject.objects.delete)
```

Comment:
`Manager.delete()` normally doesn't exist - see [the docs](https://docs.djangoproject.com/en/2.2/topics/db/managers/#creating-a-manager-with-queryset-methods)

```suggestion
        self.assertRaises(TypeError, BesluitInformatieObject.objects.all().delete)
```